parts/linux/cloud-init/artifacts/cse_helpers.sh

```diff
@@ -289,4 +290,28 @@ isARM64() {
         echo 0
     fi
 }
+
+logs_to_events() {
+    task=$1; shift
```

Comment:
do we see value in allowing the task name to always be defined. Right now all the cse_main references are AKS.CSE._func describe_

If we see value in being able to change AKS.CSE to whatever we want I can leave this as is. Or only accept a _func describe_ and have this logs_to_events append the _func describe_ to hard coded AKS.CSE in the func